app/agents/voice/automatic/stt/__init__.py

```
@@ -247,6 +252,17 @@ def get_stt_service(voice_name: Optional[str] = None):
             params=soniox_params,
             vad_force_turn_endpoint=SONIOX_VAD_FORCE_TURN_ENDPOINT,
         )
+
+    elif STT_PROVIDER == "sarvam":
+        if not SARVAM_API_KEY:
+            raise ValueError("SARVAM_API_KEY is required when STT_PROVIDER=sarvam")
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)